src/noise.ts

```diff
@@ -32,7 +32,7 @@ export class Noise implements INoiseConnection {
   public protocol = '/noise'
   public crypto: ICryptoInterface
 
-  private readonly prologue = new Uint8Array(0)
+  private readonly prologue: Uint8Array;
```

Comment:
Can you run the linter please, chances are it'll fail on the semicolon.

Reply:
It did complain about semicolons, but not here.

````
/home/lbl/code/js-libp2p-noise/src/noise.ts:57:36: Extra semicolon. [Error/@typescript-eslint/semi]
/home/lbl/code/js-libp2p-noise/src/noise.ts:59:40: Extra semicolon. [Error/@typescript-eslint/semi]
````

In any case, I'm removing this semicolon since none of the other members next to it have semicolons. No reason for this one to be special (other than my editor formats-on-save).